AppController/djinn.rb

```diff
@@ -111,6 +111,10 @@
 APPSCALE_APP_LOGROTATE = 'appscale-app-logrotate.conf'
 
 
+# The location of the appscale-upload-app script from appscale-tools.
+UPLOAD_APP_SCRIPT = '/usr/local/bin/appscale-upload-app'
```

Comment:
Can that path be dynamically discovered? What would be the case in other distros?